src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,249 @@
+<script lang="ts">
+  import type { QuizData, TeamMatch } from '$lib/constants/quiz';
+  import LeftArrow from '$lib/components/icons/left-arrow.svelte';
+  import RightArrow from '$lib/components/icons/right-arrow.svelte';
+  import MoreInfo from './more-info.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
+  let response: TeamMatch[] = [];
```

Comment:
Since the variable `response` holds a list of responses, we could name it something plural e.g. `responses` or `choices`.

Reply:
Done